packages/web/src/generators/application/schema.d.ts

```diff
@@ -1,4 +1,4 @@
-import { Linter } from '@nrwl/linter';
+import type { Linter } from '@nrwl/linter';
 
```

Comment:
Should the `linter` here be marked with `none` as well?

`linter?: Linter | 'none'`

Reply:
`'none'` is already in `Linter`, we just never updated web :)